lacework-agent/values.schema.json

```diff
@@ -93,17 +114,85 @@
                     ]
                 },
                 "labels": {
-                    "type": "object",
                     "additionalProperties": {
-                        "type": "string"
-                    }
+                        "type": [
+                            "string",
+                            "null"
+                        ]
+                    },
+                    "description": "Map of string keys and values that can be used to organize and categorize (scope and select) objects. May match selectors of replication controllers and services. More info: http://kubernetes.io/docs/user-guide/labels",
+                    "type": [
+                        "object",
+                        "null"
+                    ]
                 },
                 "env": {
                     "type": [
                         "string",
                         "null"
                     ]
                 },
+                "fim": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": [
+                        "coolingPeriod",
+                        "crawlInterval",
+                        "enable",
+                        "fileIgnore",
+                        "filePath",
+                        "noAtime",
+                        "runAt"
+                    ],
```

Comment:
The four fim properties (coolingPeriod, crawlInterval, noAtime, runAt) should either:
* Have null removed from their types and a non-null default value added to values.yaml, or
* Should be removed from the list of required properties